ENIGMAsystem/SHELL/Universal_System/Extensions/DataStructures/data_structures.cpp

```diff
@@ -1503,7 +1503,7 @@ variant ds_priority_find_min(const unsigned int id)
 {
   //Returns the value with the smallest priority but does not delete it from the priority queue
   multimap<variant, variant>::iterator it = ds_prioritys[id].begin(), it_check;
-  if (it == ds_prioritys[id].end()) {return 0;}
+  if (it == ds_prioritys[id].end()) {return variant();}
```

Comment:
I'd have deleted those {} while I was at it, but ok.

Reply:
Oops I missed this. That's ok, I haven't really wanted to mess with the data structures extensions at all. They have worked overall pretty good and I don't want to jinx that (hence writing CI tests for them), so I don't have the inclination to change more than I need to.